packages/botonic-cli/tests/botonic-api-service.test.ts

```diff
@@ -0,0 +1,17 @@
+import { join } from 'path'
+import { chdir, cwd } from 'process'
+
+import { BotonicAPIService } from '../src/botonic-api-service'
+
+const workingDirectory = cwd()
```

Comment:
can't we use one of the projects at botonic-examples (or all of them)? otherwise we're adding a new project to maintain

Reply:
Sure, taking a look on it! Good point 👍